fixtures/cmr.uat.earthdata.nasa.gov-443/165876324053364067

```diff
@@ -0,0 +1,30 @@
+POST /search/granules.json
```

Comment:
I'm assuming there are a lot of fixtures for CMR calls that should be deleted as a result of this change.

Reply:
Cleaned up